django/forms/widgets.py

```diff
@@ -594,7 +594,7 @@ def create_option(self, name, value, label, selected, index, subindex=None, attr
             option_attrs['id'] = self.id_for_label(option_attrs['id'], index)
         return {
             'name': name,
-            'value': value,
+            'value': force_text(value),
```

Comment:
Makes sense. I chose `force_text()` here to match the [earlier use](https://github.com/django/django/blob/b7dc9980871839cbd55e43c915d48388ef5e74ec/django/forms/widgets.py#L573) in `ChoiceWidget`. Do you think it would be best to change this other usage in this PR or in a separate commit & PR?